docs/asyncapi.yaml

```diff
@@ -260,4 +195,167 @@ components:
         value:
           type: string
           description: The reduction value
-          example: 75.00
+          example: '75.00'
+    FundingTotal:
+      additionalProperties: false
+      type: object
+      required:
```

Comment:
Level and Rate not required?

Reply:
https://github.com/DEFRA/ffc-pay-statement-generator/pull/33#discussion_r1104334837